bootstrap/pkg/utils/k8utils.go

```diff
@@ -38,10 +44,152 @@ import (
 const RecommendedConfigPathEnvVar = "KUBECONFIG"
 
 const (
-	yamlSeparator = "---"
+	yamlSeparator   = "---"
+	maxRetries      = 5
+	backoffInterval = 5 * time.Second
 )
 
-// TODO COPIED from bootstrap/app/k8sUtil.go. Need to merge.
+func getRESTClient(config *rest.Config, group string, version string) (*rest.RESTClient, error) {
+	c := rest.CopyConfig(config)
+	c.GroupVersion = &schema.GroupVersion{
+		Group:   group,
+		Version: version,
+	}
+	c.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
+	if group == "" {
+		c.APIPath = "/api"
+	} else {
+		c.APIPath = "/apis"
+	}
+	return rest.RESTClientFor(c)
+}
+
+func getResource(mapping *meta.RESTMapping, config *rest.Config, group string,
+	version string, namespace string, name string) error {
+	restClient, err := getRESTClient(config, group, version)
+	if err != nil {
+		return fmt.Errorf("getResource error: %v", err)
+	}
+
+	_, err = restClient.
```

Comment:
get should return the response?

Reply:
currently we don't use the response.  added a TODO in case we need to do it.